### ISSUE TITLE
Joystick animation not working

### ISSUE THREAD

**Issue by edhom** [rive-app/rive-flutter#331]
### Description

It seems like joystick animations don't work as expected for us in Flutter runtime. In the animation below, the water should move up as the key moves in... Are we missing anything?

### Steps To Reproduce

<details>
<summary>Demo App</summary>

```
void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Joystick Demo',
      home: Scaffold(
        body: Container(
          color: Colors.black87,
          child: const Center(
            child: RiveAnimation.asset(
              'joysticks.riv',
              fit: BoxFit.contain,
              stateMachines: ['State Machine'],
            ),
          ),
        ),
      ),
    );
  }
}
```
</details>

#### Source `.riv`/`.rev` file

https://editor.rive.app/file/key-joystick/459047

[joystick_demo.zip](https://github.com/rive-app/rive-flutter/files/12024624/joystick_demo.zip)

### Actual behavior

https://github.com/rive-app/rive-flutter/assets/47747733/674a7b0c-ee07-4d9d-a786-dd9fb9b2b598

### Expected behavior

Water moves up

### Device & Versions (please complete the following information)

- Rive: ^0.11.3

- Device: iOS Simulator, Web
- OS: iOS 16.0
```
Flutter 3.10.5 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 796c8ef792 (4 weeks ago) • 2023-06-13 15:51:02 -0700
Engine • revision 45f6e00911
Tools • Dart 3.0.5 • DevTools 2.23.1
```

**Comment by HayesGordon** [comment]
Thanks for opening the issue. We're looking into it

**Comment by HayesGordon** [comment]
Fixed in v0.11.5
